1.9/api/master-routes.md

```diff
@@ -0,0 +1,16 @@
+---
+post_title: Master Routes
+menu_order: 1
+---
+
+Admin Router runs on DC/OS master nodes and exposes the following API routes, grouped by the component to which they proxy or relate.
+
+Admin Router listens on port `80` (http) and `443` (https).
```

Comment:
Admin Router, the component service, runs on every master node. So there's not really one official admin router in this context...